includes/class.llms.install.php

```diff
@@ -286,7 +286,7 @@ public static function get_pages() {
 					'docs_url'     => 'https://lifterlms.com/docs/checkout-page/?utm_source=LifterLMS%20Plugin&utm_campaign=Plugin%20to%20Sale&utm_medium=Wizard&utm_content=LifterLMS%20Checkout%20Page',
 				),
 				array(
-					'content'      => '[lifterlms_my_account]',
+					'content'      => '[lifterlms_my_account layout="columns"]',
```

Comment:
The default is stacked, but the thought is to make it columns for new installs going forward?

Reply:
Yes - I have a conversation open on this in Slack. If we default to columns we break sites with themes (Kadence) that have custom styling for LifterLMS Student Dashboard. The admin would have to go in an explicitly set to "stacked" on an existing site. We can decide the best path in the Slack conversation.

Reply:
I've updated to default to columns. We can add to our CSS adjustments to Kadence when we release this.